wiki/UI-component/Video.md

```diff
@@ -39,6 +39,147 @@ The [[Video]] component utilizes the embeddable Kaltura player <sup>[[r2](#resou
+
+### CTA - Text
+
+![CTA - Text](https://user-images.githubusercontent.com/15144993/123468111-523aff00-d5bf-11eb-8ef1-2f05c0de29c6.png)
+
+| ID  | Data name     | Field |
+| --- | ------------- | ----- |
+| 1   | Name | CTA text |
+| 2   | Duration | appended after CTA text in parenthesis |
+
+Interactions
+
+| ID  | Element name | Event | Behavior |
+| --- | ------------ | ----- | -------- |
+| 1   | CTA - Text | On Click or On Enter key of link | Open video in LightBoxMediaViewer. |
+
+
+### CTA - Button
+
+![CTA - Button](https://user-images.githubusercontent.com/15144993/123468110-523aff00-d5bf-11eb-9b1d-0c65f082d482.png)
+
+| ID  | Data name     | Field |
+| --- | ------------- | ----- |
+| 1   | Name | Button label |
+| 2   | Duration | appended after Button label in parenthesis |
+
+Interactions
+
+| ID  | Element name | Event | Behavior |
+| --- | ------------ | ----- | -------- |
+| 1   | CTA - Button | On Click or On Enter key of button | Open video in LightBoxMediaViewer. |
+
+
+### CTA - Card
+
+![CTA - Card](https://user-images.githubusercontent.com/15144993/123468108-51a26880-d5bf-11eb-93a0-c23ef1c8216b.png)
+
+| ID  | Data name     | Field |
+| --- | ------------- | ----- |
+| 1   | Name | Card text |
+| 2   | Description | Card body copy |
+| 3   | Duration | CTA text | No |
+| 4   | Poster image | Image |
+
+Interactions
+
+| ID  | Element name | Event | Behavior |
+| --- | ------------ | ----- | -------- |
+| 1   | CTA - Card   | On Click or On Enter key of card | Open video in LightBoxMediaViewer. |
+
+
+### Lightbox media viewer
+
+![Lightbox](https://user-images.githubusercontent.com/15144993/123468106-5109d200-d5bf-11eb-8556-828dc20b5ccd.png)
+
+| ID  | Data name     | Field |
+| --- | ------------- | ----- |
+| 1   | Name | Lightbox headline |
+| 2   | Description | Lightbox body copy |
+| 3   | Poster image | Image |
+
+Interactions
+
+| ID  | Element name | Event | Behavior |
+| --- | ------------ | ----- | -------- |
+| 1   | LightboxMediaViewer | On open | Begins playing video |
+| 2   | Kaltura embedded video | See [standard Kaltura Player functionality](http://player.kaltura.com/docs/main) | Contains all available functionality of the standard Kaltura Player |
+| 3   | LightboxMediaViewer | On close | Stops playing video |
+
+## 3. Metrics
+
+The following events will need to be tracked with IBM's IDA Stats event tracking
+
+| Event name | Event code | Description |
+| --- | --- | --- |
+| launched | 0 | Video launched from inline or text link state and started playing |
+| paused | 1 | Video playback has been paused |
+| played | 2 | Video playback has started (should trigger playerState “launched” if the currentTime is 0) |
+| ended | 3 | Video playback has reached the end |
```

Comment:
I also kind of wonder about the relevance of this anymore. All of our components should be enabled for metric tracking. I wonder if this should be more along the lines of available events instead of what Google analytics is doing with those events?

Reply:
They made no sense to me! Would love to remove. Just weren't sure what it's for.